src/components/dropDown/DropDown.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useState } from 'react';
+import { ScrollView, StyleSheet, Text, View, ViewStyle } from 'react-native';
+import { List } from 'react-native-paper';
+import { useTheme } from '../../styles/themes';
+import { Icon } from '../icons/Icon';
+
+export type DropDownOption = {
+    name: string;
+    value: any;
```

Comment:
What is the value used for ?

Reply:
I talked with Valentin about that, we are not sure what with object will contain, so decided to leave it like this. Do you have suggestions what to put here?